### ISSUE TITLE
add location to link to external data resource

### ISSUE THREAD

**Issue by mbiddle-bcodmo** [BCODMO/Ocean-Data-Ontology#16]
We've come across a few submissions that are, in essence, links to other data resources. For example, NCBI Project numbers. Right now we are doing a variety of things with those links. Linking in the Data url, providing a link in the metadata description, or creating an object which contains the link. 

@DanieK, @adyork, @whshannon, and myself had this discussion earlier today and wanted to explore the possibilities. One thought would be to create a specified location in the metadata record to record this information. Thoughts?

**Comment by ashepherd** [comment]
Yeah nice! Do we get any useful metadata related to each NCBI link that if was searchable could be used to aid discovery? Wondering how similar this issue might be to say datasets of images and video (a tabular file of metadata including pink off to the data resources)

**Comment by adyork** [comment]
We discussed that a bit.

In cases where they actually do give us a dataset with the accessions and related data that is appropriate for BCO-DMO to serve, we can serve it as a normal dataset.  For example when they give us sample date, time, lat, lon, environmental data, nutrients, or experimental information. 

In those cases we discussed if it was appropriate to add the bioproject as a "related resource" to the dataset.

However, I think the main driver for this discussion for an "external dataURL" is when we are only given the bioproject accession number and no tabular data that would make sense for us to serve.

It could be possible for us to go through the hoops of querying information at ncbi and creating a redundant copy of the data that we could serve, but there is no value added by BCO-DMO in that case.

Does that clear up why we are asking about what to do in cases where we just have a link to ncbi data but no tabular data?

**Comment by adyork** [comment]
Also...not sure if this is helpful but I just saw that all the SRA data is available through some AWS and GCP.  https://ncbiinsights.ncbi.nlm.nih.gov/2020/02/24/sra-cloud/
access info https://www.ncbi.nlm.nih.gov/sra/docs/sra-cloud/

Though that doesn't help for our GenBank stuff.
